src/System.Security.Cryptography.X509Certificates/src/System/Security/Cryptography/X509Certificates/Pkcs10CertificationRequestInfo.cs

```diff
@@ -63,11 +63,14 @@ internal byte[] ToPkcs10Request(X509SignatureGenerator signatureGenerator, HashA
                 Version = 0,
                 Subject = this.Subject.RawData,
                 SubjectPublicKeyInfo = spki,
-                Attributes = Attributes.Select(e => new X501AttributeAsn(e)).ToArray(),
+                Attributes = Attributes.Select(a => new X501AttributeAsn(a)).ToArray(),
```

Comment:
While I'm not necessarily arguing with the meat of the change, it doesn't seem like it was necessary or relevant?

Reply:
I originally copied the code from a different place that used `X509ExtensionAsn`. During the conversion I changed this line couple of times and in the end it ended up almost unchanged except for this little bit. It bothered me because `e` is for `[e]xtension` and `a` is for `[a]ttribute`.